test/functional/terminal/scrollback_spec.lua

```diff
@@ -448,7 +448,7 @@ describe("'scrollback' option", function()
     if iswin() then
       feed_data('for /L %I in (1,1,40) do @(echo line%I)\r')
     else
-      feed_data('for i in $(seq 1 40); do echo "line$i"; done\n')
+      feed_data('awk "BEGIN{for(n=1;n<=40;n++) print \\\"line\\\" n}"\n')
     end
```

Comment:
i think we can use `shell-test` now, for all platforms including Windows. 

Example: https://github.com/neovim/neovim/blob/6b45e12d67ec01c8a/test/functional/ui/searchhl_spec.lua#L150

    feed([[:terminal "]]..nvim_dir..[[/shell-test" REP 5000 foo<cr>]])

Reply:
```
Expected (anywhere): "line30                        " 
Actual:                             
  |test" REP 30 line           < |                                                                        
  |/bin/sh: :terminal: not found |                                                                        
  |myhost$ {1: }                   |                                                                    
  |                              |                   
  |                              |
  |                              |                                                                        
  |{3:-- TERMINAL --}                |
```

It might be easier for you to do this part, as I don't know anything about lua or your test suite ;)

Reply:
I also tried killing the `:terminal` prefix, but it looks to me like we are expecting to execute a ex mode command, but instead pass it to the shell.

Reply:
Also noticed this stuff in that same test file:
```
  local function set_fake_shell()      
    -- shell-test.c is a fake shell that prints its arguments and exits.
    nvim('set_option', 'shell', nvim_dir..'/shell-test')
    nvim('set_option', 'shellcmdflag', 'EXE')          
  end
```

Not sure if that interferes?